orte/runtime/orte_globals.c

```diff
@@ -889,6 +889,9 @@ static void orte_attr_des(orte_attribute_t *p)
         OBJ_DESTRUCT(&p->data.buf);
     } else if (OPAL_STRING == p->type) {
         free(p->data.string);
+    } else if (OPAL_ENVAR == p->type) {
+        free(p->data.envar.envar);
+        free(p->data.envar.value);
```

Comment:
Hmmm...this is fine for now. We should probably get rid of that data type as it was created solely to support the old opal/pmix abstraction framework, and thus is no longer needed.